### DISCUSSION TITLE
Can we use fast-formula-parser to alternative this one?

### DISCUSSION THREAD

**Discussion by proc01** [handsontable/hyperformula#1129] [General]
Because the hyperformula is under GPLv3, so we can not have in our commercial product.
Can we have another alternative this one?

**Comment by sequba** [comment]
We do offer a commercial license for HyperFormula. Write to us at [sales@handsontable.com](mailto:sales@handsontable.com) or submit your inquiry through the [contact form](https://handsontable.com/contact?category=request_for_quotation).

**Reply by proc01** [reply]
thanks
